frontend/src/index.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import playerReducer from './redux/reducers';
 import './index.css';
 import Nav from './Nav';
 import * as serviceWorker from './serviceWorker';
 
+const store = createStore(playerReducer)
```

Comment:
suggestion: try adding `(playerReducer, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__() )` here. This enables the chrome dev tools extension. If you then add it to chrome, you get a nice GUI to view your state and how it is changing